release-team/role-handbooks/bug-triage/README.md

```diff
@@ -1,56 +1,81 @@
+This is the best stage to get involved with any automation work that can ease the workload of later stages.
+
+As this stage lasts nearly 2 months up until Code Freeze, you can use this time to familiarize yourself with the _major enhancements and fixes_ planned by each SIG for this release, so that you can have context in advance of when you will need to identify incoming bugs as being associated with a work focus in the current release. This can greatly help in focused communication with the relevant SIG leads, as in later stages there can be an elevated urgency to fix related critical bugs and deliver new features in time.
+
+It is also a good time to interact with the Enhancements Lead and CI Signal Lead to understand any early concerns they might have, as the release team's risk management comes as much from this proactive collaboration more as from the Bug Triage lead reacting to incoming issues and PRs.
 
-This is also a good time to do any work on automation that you plan to do.  You can also get started early on the pre-code-slush work.
 
 ### Sample Searches
 
-* Open Enhancement v1.11 Issues:
-  * `is:issue is:open milestone:v1.11 repo:kubernetes/enhancements` [k/enhancements repo milestone enhancements](https://github.com/kubernetes/enhancements/issues?q=is%3Aopen+is%3Aissue+milestone%3Av1.11)
-  * `is:issue is:open milestone:v1.11 label:kind/feature repo:kubernetes/kubernetes` [k/k repo milestone enhancements](https://github.com/kubernetes/kubernetes/issues?q=is%3Aopen+is%3Aissue+milestone%3Av1.11+label%3Akind%2Ffeature)
-  * `is:issue is:open milestone:v1.11 label:kind/feature org:kubernetes` [k org-wide milestone enhancements](https://github.com/search?q=org%3Akubernetes+is%3Aissue+is%3Aopen+milestone%3Av1.11+label%3Akind%2Ffeature)
-* Issues in the v1.11 milestone which haven't been updated in a while: `is:open is:issue milestone:v1.11 updated:<2018-05-01 repo:kubernetes/kubernetes` [k/k old issues in milestone](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+milestone%3Av1.11+updated%3A%3C2018-05-01+repo%3Akubernetes%2Fkubernetes)
-* Issues in the milestone with no SIG or Kind: **code/automation needed**
-* Enhancement Issues with no PR: **code/automation needed**
+* Issues in the v1.14 milestone which haven't been updated in a while: `[is:open is:issue milestone:v1.14 updated:<2018-10-01 repo:kubernetes/kubernetes](https://github.com/kubernetes/kubernetes/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+milestone%3Av1.11+updated%3A%3C2018-10-01+repo%3Akubernetes%2Fkubernetes)`
+
```

Comment:
Did you intend to move the closing back tick to the end of the line on these two urls?

Reply:
good catch, fixed